test/specs/rules/require-arrow-functions.js

```diff
@@ -0,0 +1,44 @@
+var Checker = require('../../../lib/checker');
+var assert = require('assert');
+
+describe('rules/require-arrow-functions', function() {
+    var checker;
+
+    beforeEach(function() {
+        checker = new Checker();
+        checker.registerDefaultRules();
+        checker.configure({ requireArrowFunctions: true, esnext: true });
+    });
+
+    it('should report use of function expression in declaration', function() {
```

Comment:
For sanity, can we add a test for function expressions assignments as object properties?

i.e., 

```js
var foo = {};
foo.bar = function() {};
```

and

```js
var foo = {
  bar: function() {}
};
```

Named function expressions